src/components/Header.js

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
+import { images } from "./images/image";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+  //change parent's user state using props
+  const handleClick = () => {
+    props.onClick();
+  };
+
+  return (
+    <Profile onClick={handleClick}>
+      {/* if user value is false */}
+      {/* I hate this code!!!!!! I want simpler code... */}
+      {!props.user && (
+        <>
+          <img src={images.COOL} />
+          <div>
+            <h2 style={{ marginBottom: "0" }}>정쿨</h2>
+            <br />
+            현재 활동중
+          </div>
+        </>
+      )}
+      {/* if user value is true */}
+      {props.user && (
+        <>
+          <img src={images.EUNKO} />
+          <div>
+            <h2 style={{ marginBottom: "0" }}>고은</h2>
+            <br />
+            현재 활동중
+          </div>
+        </>
+      )}
+    </Profile>
+  );
 }
-const Wrapper = styled.div`
-	background-color: black;
-	color: white;
+
+const Profile = styled.div`
+  background-color: white;
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  div {
+    margin-left: 10px;
+  }
+
+  img {
+    border-radius: 20px;
+    width: 100px;
+    height: 100px;
+  }
```

Comment:
이것도 따로 styled-component로 빼주시면 좋을 것 같습니다..!ㅜㅜ

```suggestion
const Img = styled.img`
	border-radius: 20px;
	width: 100px;
	height: 100px;
`;
```

Reply:
넵 수정하겠습니다~!